src/components/message/view.js

```diff
@@ -160,7 +170,7 @@ const Action = (props: ActionProps) => {
             tipText={`Reply`}
             tipLocation={'top'}
             size={24}
-            onClick={replyToMessage}
+            onMouseDown={replyToMessage} // onClick deselects selected text
```

Comment:
@mxstbr @brianlovin `onClick` deselects selected text, so we always get `null`. so, i had to use `onMouseDown`. what do you think ? is there any better way ?